### PR TITLE
add rfc merge procedure

### PR DESCRIPTION
**Pull request by nikomatsakis**
cc @steveklabnik

### CODE REVIEW

File: rfc-merge-procedure.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+**Unresolved questions:**
+
+XXX --- list all the "unresolved questions" found in the RFC
+        to ensure they are not forgotten
+```   
+
+Add the following labels to the issue:
+
+- `B-rfc-approved`
+- `B-unstable`
```
Comment (aturon) [inline rfc-merge-procedure.md]: I think `B-unstable` is only supposed to be applied when the impl has landed.

File: rfc-merge-procedure.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: RFC Merge Procedure
+---
+
+Once an RFC has been accepted (i.e., the final comment period is
```
Comment (aturon) [inline rfc-merge-procedure.md]: Probably good to clarify that this is after an FCP *where no serious new issues/arguments were raised*.

**Comment by steveklabnik** [comment]
🎊

**Comment by steveklabnik** [comment]
Oh no, GitHub didn't show me @aturon 's comments.

File: rfc-merge-procedure.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+the most appropriate team).
+
+```
+This is a tracking issue for the RFC "XXX" (rust-lang/rfcs#NNN).
+
+**Steps:**
+
+- [ ] Implement the RFC (cc @rust-lang/XXX -- can anyone write up mentoring instructions?)
+- [ ] Adjust documentation ([see instructions on forge][doc-guide])
+- [ ] Stabilization PR ([see instructions on forge][stabilization-guide])
```
Comment (eddyb) [inline rfc-merge-procedure.md]: How are the links here supposed to work?

**Comment by eddyb** [comment]
There's no mention of what the title of the tracking issue should be.

File: rfc-merge-procedure.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+### Step 3: Leave a comment
+
+Leave a final comment on the PR directing everyone to the tracking
+issue. Something like this, but feel free to add your own personal
+flavor (and change the team):
+
+```
+**Huzzah!** The @rust-lang/lang team has decided **to accept** this RFC. 
+
+To track further discussion, subscribe to the tracking issue here: rust-lang/rust#41517
```
Comment (eddyb) [inline rfc-merge-procedure.md]: This uses a specific number instead of a placeholder.

**Pull request merged**